commit 511d99b7745a6ce87a208bdbf69e631f1f136d53
Author: Luca Beltrame <lbeltrame@kde.org>
Date:   Sun Jul 21 09:07:44 2019 +0200

    Fix build with Qt 5.13
    
    The QDate include is no longer included transitively.

diff --git a/src/plugins/forms/widgets/kexidbdatepicker.cpp b/src/plugins/forms/widgets/kexidbdatepicker.cpp
index 1e0692d5b..92a5c880b 100644
--- a/src/plugins/forms/widgets/kexidbdatepicker.cpp
+++ b/src/plugins/forms/widgets/kexidbdatepicker.cpp
@@ -22,6 +22,7 @@
 #include "kexidbdatepicker.h"
 #include <kexiutils/utils.h>
 
+#include <QDate>
 #include <QLineEdit>
 #include <QInputMethodEvent>
 
